src/main/java/io/leonis/torch/window/RxPanel.java

```diff
@@ -0,0 +1,37 @@
+package io.leonis.torch.window;
+
+import com.googlecode.lanterna.gui2.*;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Schedulers;
+
+/**
+ * @author jeroen.dejong
+ */
+public final class RxPanel extends Panel {
+
+  public RxPanel() {
+    super();
+  }
+
+  public RxPanel addComponent(final Publisher<? extends Component> publisher) {
+    Flux.from(publisher)
+        .subscribeOn(Schedulers.single())
+        .transform(componentFlux -> {
```

Comment:
maybe just solve this in two seperate fluxes; of which one flux will simply subscribes for the first; and the second has the buffer(2,1).